src/reducers/playListReducer.js

```diff
@@ -49,6 +50,26 @@ export const playListReducer = (playListState, { type, payload }) => {
             : playlist
         ),
       };
+    case "ADD_TO_LIKED_VIDEOS":
+      return {
+        ...playListState,
+        likedVideos: payload,
+      };
+    case "REMOVE_FROM_LIKED_VIDEOS":
```

Comment:
if possible make a file and assign these strings to a variable and use variable here to avoid spelling mistakes and make code easy to manage

Reply:
done